pkg/render/node.go

```diff
@@ -628,6 +629,10 @@ func (c *nodeComponent) createCalicoPluginConfig() map[string]interface{} {
 
 	apiRoot := c.cfg.K8sServiceEp.CNIAPIRoot()
 
+	LinuxpolicySetupTimeoutSeconds := int32(0)
```

Comment:
This should be a private variable, and `Policy` should be capitalized

`linuxPolicySetupTimeoutSeconds`